api/specs/web-server/_resource_usage.py

```diff
@@ -232,46 +163,37 @@ async def update_pricing_plan(
     tags=["admin"],
 )
 async def get_pricing_unit(
-    pricing_plan_id: PricingPlanId, pricing_unit_id: PricingUnitId
+    _p: Annotated[PricingUnitGetPathParams, Depends()],
 ):
     ...
 
 
-assert_handler_signature_against_model(get_pricing_unit, _GetPricingUnitPathParams)
-
-
 @router.post(
     "/admin/pricing-plans/{pricing_plan_id}/pricing-units",
     response_model=Envelope[PricingUnitAdminGet],
     summary="Create pricing unit",
     tags=["admin"],
 )
 async def create_pricing_unit(
-    pricing_plan_id: PricingPlanId, body: CreatePricingUnitBodyParams
+    _p: Annotated[PricingPlanGetPathParams, Depends()],
+    _b: CreatePricingUnitBodyParams,
 ):
     ...
 
 
-assert_handler_signature_against_model(create_pricing_unit, _GetPricingPlanPathParams)
-
-
 @router.put(
     "/admin/pricing-plans/{pricing_plan_id}/pricing-units/{pricing_unit_id}",
     response_model=Envelope[PricingUnitAdminGet],
     summary="Update detail information about pricing plan",
     tags=["admin"],
 )
 async def update_pricing_unit(
-    pricing_plan_id: PricingPlanId,
-    pricing_unit_id: PricingUnitId,
-    body: UpdatePricingUnitBodyParams,
+    _p: Annotated[PricingUnitGetPathParams, Depends()],
+    _b: UpdatePricingUnitBodyParams,
 ):
```

Comment:
minor: actually. I think writing _path, _query and _body is not so long and goes along using long option names ;) but that's your call.

Reply:
I also like the long options more